### ISSUE TITLE
Add ability to cancel/blacklist download

### ISSUE THREAD

**Issue by MarekPraha** [shadowmoose/RedditDownloader#143]
For some reason, my downloader keeps downloading the same links which are bad because they link to Twitch streamers and youtube channels and downloader starts downloading them.

I would like to have a blacklist of downloads or the ability to cancel downloads.

**Comment by shadowmoose** [comment]
I agree that this is a good idea. The latest RMD release supports disabling retries, but that doesn't always solve the issue.
